config/global.js

```diff
@@ -10,11 +10,13 @@ const {
 	SC_DOMAIN = 'localhost',
 	SC_THEME = 'default',
 	REDIS_URI,
-	REQUEST_TIMEOUT = 15000, // 15 sec
+	REQUEST_TIMEOUT = 5000, // 5 sec
```

Comment:
```suggestion
	REQUEST_TIMEOUT_MILLIS = 5000, // 5 sec
```
Is it possible to add the millis into the name?

Reply:
i rename it

Reply:
done